packages/h5p-server/test/ContentUserDataManager.test.ts

```diff
@@ -0,0 +1,211 @@
+import ContentUserDataManager from '../src/ContentUserDataManager';
+
+import MockContentUserDataStorage from './__mocks__/ContentUserDataStorage';
+
+import User from './User';
+
+describe('ContentUserDataManager', () => {
```

Comment:
The tests here mostly check if function of the storage class are called. I think it would be more useful if tests checked actual functionality, like saving a state and retrieving it later. Just checking if calls are passed to the storage class might lead to a false sense of secruity as it produces good line coverage, but doesn't check any real functionality or logic.

Reply:
I agree with that, but we should think about separating two tests:
unit-test which solely tests the "unit"  and integration tests. If you add an actual storage class you also test the functionality of the storage class. If you want a pure unit test you have to mock everything else.
I will add an integration test as a separate test.